python/cuml/test/test_metrics.py

```diff
@@ -154,3 +158,49 @@ def test_rand_index_score(name, nrows):
     cu_score_using_sk = sk_ars(y, cp.asnumpy(cu_y_pred))
 
     assert array_equal(cu_score, cu_score_using_sk)
+
+
+@pytest.mark.parametrize('n_samples', [50, 100, 2000])
```

Comment:
why this sizes? seems like 50, 2000 would be enough, but what does 2000 test that 50 doesn't? Also could you add stress_param as well to have stress tests?

Reply:
I agree, I removed `[100, 2000]` and instead I've added a stress tests with random values.